pkg/products/observability/prometheusRules.go

```diff
@@ -341,6 +341,23 @@ func (r *Reconciler) newAlertsReconciler(logger l.Logger, installType string) re
 				},
 			},
 		},
+		{
+			AlertName: fmt.Sprintf("%s-missing-metrics", installationName),
+			Namespace: namespace,
+			GroupName: fmt.Sprintf("%s-general.rules", installationName),
+			Rules: []monitoringv1.Rule{
+				{
+					Alert: fmt.Sprintf("%sCriticalMetricsMissing", strings.ToUpper(installationName)),
+					Annotations: map[string]string{
+						"sop_url": resources.SopUrlCriticalMetricsMissing,
+						"message": fmt.Sprintf("%s metrics have been missing for more than 5 minutes", strings.ToUpper(installationName)),
+					},
+					Expr:   intstr.FromString(`(` + installationName + `_version{version=~".*", to_version=~""} > 0) * on(pod) group_left(` + installationName + `_custom_domain)(absent(` + installationName + `_custom_domain)) or absent(` + installationName + `_version)`),
```

Comment:
is the empty space before the installationName necessary?

Reply:
nvm! :man_facepalming:

Reply:
Although I would keep it the same as all other alerts; using the fmt.Sprintf if you can like here:
https://github.com/integr8ly/integreatly-operator/pull/2802/files#diff-7139745d8ee64821c1d2176216e15673c888f7cd064df149a05527f34027efd4R81 to keep consistent

Reply:
I don't mind changing it but most of the alerts above it don't use the fmt.Sprintf in the experession

Reply:
@den-rgb most of them (if not all) do use fmt.Sprintf if we need to pass in a variable to the query. I'd like this alert to stay consistent with others if possible.

Reply:
also, looking at only the file you are changing, notice how the other alerts are using it; it is not used when we don't need to take in a variable

Reply:
and just to re-iterate; this is not a big deal and the only reason I've picked that up is that I made a mistake when reading it for the first time, due to it being constructed in a different way than others.